nextcloud/android
# Fix AutoUpload notification

fix #3163 

- fix not working detection of already defined synced folders
- delete not used (and wrong) function

Problem was that passing everything into selection did not worked.
Also this is error prone to sql injection (yes, very theoratical with folder names), but changing it to use correct parameter solves both.

Steps to test:
- have images in camera folder
- make sure that download folder is not in media scanner (delete images from download in gallery app)
- clean installation of app
- set up camera folder
- take image -> no notification
- delete all files in camera folder
- take image -> no notification
- download image e.g. with Firefox --> see notification

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>
